integration/relayer/cmd_relayer_test.go

```diff
@@ -604,9 +458,4 @@ func TestBlogIBC(t *testing.T) {
 		),
 	)
 	env.Must(env.Exec("check ibc balance", steps, envtest.ExecRetry()))
-
-	// TODO test ibc using the blog post methods:
```

Comment:
It has already been tested in  the apps repository

Reply:
This one makes sense.